api/admin.py

```diff
@@ -2,6 +2,6 @@
 Not currently in use.
 """
 
-from django.contrib import admin
+# from django.contrib import admin
```

Comment:
Ahhhh I know why these are commented out, ruff was upset about unused imports, but I wanted to keep these here for if/when I do need them.  Add a comment saying this to each of these (api/admin, api/tests, Frontend/tests so far).

Reply:
Resolved by 1f0c591